modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/surface/Surface.js

```diff
@@ -125,7 +130,7 @@ class Surface extends Disposable {
 	 * @return {Element}
 	 */
 	createChild(screenId) {
-		var child = globals.document.createElement('div');
+		var child = document.createElement('div');
```

Comment:
Mixed `var`/`const`/`let` is a bit distracting throughout all these files. Not necessary for this PR, but I'd put it on the "Boy Scout TODO List" for possible improvement.